front/poke/temporal/activities.ts

```diff
@@ -539,9 +533,9 @@ export async function deleteMembersActivity({
             `[Workspace delete] Deleting Membership ${membership.id} and user ${user.id}`
           );
           // Delete the user's files
-          await FileResource.deleteAllForUser(renderUserType(user), t);
+          await FileResource.deleteAllForUser(user.toJSON(), t);
           await membership.delete(auth, t);
-          await user.destroy({ transaction: t });
+          await user.delete();
```

Comment:
Why not passing the transaction here anymore?

Reply:
@spolu made a bunch of comments to remove transactions from the resource - so delete() doesn't have a transaction anymore